src/snowflake/vcrpy/_constant.py

```diff
@@ -4,13 +4,25 @@
 
 SNOWFLAKE_CREDENTIAL_HEADER_FIELDS = [
     "Authorization",
+    "x-amz-server-side-encryption-customer-key-MD5",
+    "x-amz-server-side-encryption-customer-key-md5",
```

Comment:
are the header fields case-sensitive -- `MD5` vs `md5`?

Reply:
Yes, I think it is case sensitive